MODULE.bazel

```diff
@@ -24,7 +24,7 @@ bazel_dep(name = "nlohmann_json", version = "3.11.3")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "protobuf", repo_name = "com_google_protobuf", version = "26.0")
 bazel_dep(name = "pybind11_bazel", version = "2.12.0")
-bazel_dep(name = "resim-python-client", version = "6c485d1450e93be66cbee85f02077ff0c2b65198")
+bazel_dep(name = "resim-python-client", version = "b39d98f0e0ee9c213d5279af7f648e81e1693a92")
```

Comment:
```suggestion
bazel_dep(name = "resim-python-client", version = "185d9c00b3d7903150a3149fd7e258d9ceae12f5")
```

Reply:
This requires another PR first then. One sec.

Reply:
This should do it! https://github.com/resim-ai/open-core/pull/189

Reply:
Done